app/main.py

```diff
@@ -1,3 +1,78 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: "Distance") -> object:
+        if isinstance(other, Distance):
+            return Distance(km=self.km + other.km)
+        else:
+            return Distance(km=self.km + other)
+
+    def __iadd__(self, other: "Distance") -> object:
+        if isinstance(other, Distance):
+            self.km = self.km + other.km
+            return self
+        else:
+            self.km = self.km + other
+            return self
```

Comment:
Rewrite this method without  else-block

Reply:
I made it, but I dont know if it is correct